constrained-voucher.mkd

```diff
@@ -662,6 +669,38 @@ If the certificate that the Registrar uses is marked as a id-kp-cmcRA certificat
 
 In summary for typical Registrar use, where a single Registrar certificate is used, then the certificate MUST have EKU of: id-kp-cmcRA, id-kp-serverAuth, id-kp-clientAuth.
 
+## MASA Extensions {#brski-masa-extensions}
+
+This document supports efficient operation of Pledges in constrained networks, by letting a Pledge optionally omit the sending of its root CA certificate in the DTLS handshake with the Registrar.
+This optimization requires a Registrar to have an alternative method to obtain this root CA certificate, which it may need for validation and logging purposes. The alternative method defined here is 
+that the Registrar can request the relevant root CA certificate(s) directly from the MASA. Although the operator of a Registrar may have already obtained such CA certificate(s) via a vendor-specific method 
+(e.g. part of sales integration), a standardized method is still needed for cases where such vendor-specific methods do not exist or are not known to the operator.
+
+### Pledge CA Certificates Request
+A Registrar, after having established a TLS connection to the MASA, MAY request the full set of root CA certificates for all Pledges that are managed in scope of this MASA, by sending a Pledge CA 
+Certificates Request. This uses a HTTPS GET with the operation MASA URI path of "/.well-known/brski/cacerts".
```

Comment:
Does this amend RFC8995, (which creates the BRSKI-MASA protocol), or does it amend RFC7030 (which defined cacerts), or ??

Reply:
@pritikin might have an opinion about whether there is any IANA Considerations here.

Reply:
It does not amend 8995, since 8995 resources are left untouched. It adds a resource which be simply be recorded in the IANA registry here: https://www.iana.org/assignments/brski-parameters/brski-parameters.xhtml#brski-well-known-uris

It also does not amend RFC7030 since it targets the BRSKI protocol and not EST. The name "cacerts" is simply borrowed and can be changed if it helps for clarity (but in principle, the well-known BRSKI and well-known EST namespaces are fully separate so we can borrow the name if we want.)

The IANA section can be added in this PR once we think this is useful to have.  (E.g. we might be able to quantify the performance gain and see if it's worth the extra text and complexity.)